glide.yaml

```diff
@@ -0,0 +1,10 @@
+package: github.com/openshift/api
+import:
+- package: k8s.io/gengo
+  version: master
```

Comment:
master is wrong, compare https://github.com/openshift-evangelists/crd-code-generation/blob/master/Gopkg.toml